### PR TITLE
chore: remove dupliation for rendering test scripts

### PR DESCRIPTION
**Pull request by LiKang6688**
Migrate from 
```
"test:local:rendering": "./test/rendering/scripts/run-rendering-test.sh",
"test:local:update:screenshots": "./test/rendering/scripts/update-screenshots.sh",
```

to

```
"test:local:rendering": "./test/rendering/scripts/run-rendering-tests.sh",
"test:local:update:screenshots": "./test/rendering/scripts/run-rendering-tests.sh '--update-snapshots'",
```

### CODE REVIEW

File: test/rendering/scripts/run-rendering-tests.sh [hunk 1]
```diff
@@ -15,6 +15,6 @@ playwrightVersion=$(npm list @playwright/test | awk '{print $2}' | cut -d "@" -f
 docker run -d --name sn-table-playwright --rm --network host -v $(pwd):/work/ -w /work/ -it mcr.microsoft.com/playwright:v$playwrightVersion-focal
 
 # The actual rendering test is running from the machine running this shell script
-docker exec sn-table-playwright /bin/sh -c "yarn test:rendering"
+docker exec sn-table-playwright /bin/sh -c "$@"
```
Comment (haxxmaxx) [inline test/rendering/scripts/run-rendering-tests.sh:18]: kind of weird to pass the yarn command and not just the flag, should be possible to just parse somehow?
`-c "yarn test:rendering $0"` where the argument is --update-snapshots

File: test/rendering/README.md [hunk 2]
```diff
@@ -19,15 +19,15 @@ Visual regression testing flow:
...
+If you've updated the UI, you need to run the update screenshots script:
 
     # Install dependencies
     yarn --frozen-lockfile
 
     # Build nebula.js visualization
     yarn build
 
-    chmod 777 ./test/rendering/scripts/update-screenshots.sh
+    chmod 777 ./test/rendering/scripts/run-rendering-tests.sh
```
Comment (haxxmaxx) [inline test/rendering/README.md:30]: you would need the argument here right?

File: test/rendering/scripts/run-rendering-tests.sh (hunk 1 above)
Comment (LiKang6688) [reply test/rendering/scripts/run-rendering-tests.sh:18]: Sure, Should be

File: test/rendering/README.md (hunk 2 above)
Comment (LiKang6688) [reply test/rendering/README.md:30]: No, here is to Change permission

File: test/rendering/scripts/run-rendering-tests.sh (hunk 1 above)
Comment (LiKang6688) [reply test/rendering/scripts/run-rendering-tests.sh]: Done

**Review by haxxmaxx — APPROVED** [review]

**Pull request merged**